<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-OU7DV3H" ignored="0" name="Tests.VerifyForm_Mensaje-Error_CodigoSis-Vacio" tests="1" failures="1" timestamp="2021-10-25T14:07:48 BOT" time="219.262" errors="0">
  <testcase name="verifyCorrectForm" time="88.06" classname="Tests.VerifyForm_4">
    <failure type="java.lang.AssertionError" message="Error expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Error expected [true] but found [false]
at Tests.VerifyForm_4.verifyCorrectForm(VerifyForm_4.java:33)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyCorrectForm -->
</testsuite> <!-- Tests.VerifyForm_Mensaje-Error_CodigoSis-Vacio -->
